Soccar [Client]/src/nl/soccar/ui/fx/models/MapUiFx.java

```diff
@@ -26,13 +25,30 @@
     private static final Image TEXTURE_MOON;
     private static final Image TEXTURE_DESERT;
     private static final Image TEXTURE_GRASS;
+    private static final float GOAL_TRIGGER_MARGIN;
+    private static final float WALL_WIDTH;
+    private static final float CORNER_SIZE;
 
     static {
         TEXTURE_MOON = new Image(DisplayConstants.LOCATION_TEXTURE_MOON);
         TEXTURE_DESERT = new Image(DisplayConstants.LOCATION_TEXTURE_DESERT);
         TEXTURE_GRASS = new Image(DisplayConstants.LOCATION_TEXTURE_GRASS);
+        GOAL_TRIGGER_MARGIN = 1.2F;
+        WALL_WIDTH = 2.0F;
+        CORNER_SIZE = 15.0F;
     }
 
+    private final GameCanvasFx canvas;

```

Comment:
Canvas staat gedeclareerd in de superklasse (DrawableFx). Kan opgehaald worden met: super.getCanvas();